_docs/Qabel-Protocol-Box.md

```diff
@@ -253,6 +253,19 @@ The plaintext of the drop message is a JSON document with the url of the DM and
 }
 ```
 
+### User to user messaging
+Users can send chat messages to their contacts. Those messages are send as drop messages with
```

Comment:
Don't know if it's too pedantic but it should be "are sent".
Apart from this :+1: